src/v2/styles/Message/Message-theme.scss

```diff
@@ -261,7 +261,7 @@
     }
   }
 
-  .str-chat__message-metadata {
+  .str-chat__message-metadata,.str-chat__custom-message-metadata {
```

Comment:
nit:
```suggestion
  .str-chat__message-metadata,
  .str-chat__custom-message-metadata {
```

Reply:
Thanks, prettier didn't run in my VSCode for some reason, I've ran it and pushed the changes it made. This one was among them :)